<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>       Select Column</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>         Select CSV</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>        Select Row</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>133</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="select_column"/>
    </item>
    <item row="1" column="3" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>113</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="QLineEdit" name="path_csv">
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>153</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="11" colspan="2">
     <widget class="QLineEdit" name="select_row"/>
    </item>
    <item row="1" column="13">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>123</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="14">
     <widget class="QTableView" name="view_dataframe"/>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>271</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="stats">
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QPushButton" name="show_data">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Show Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>89</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QPushButton" name="settings">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="10" colspan="2">
     <widget class="QPushButton" name="choose_plot">
      <property name="maximumSize">
       <size>
        <width>93</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string>Choose Plot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="12" colspan="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
